### PR TITLE
changelog: add February images update part 1

### PR DESCRIPTION
**Pull request by davlgd**
## Checklist

- [ ] My PR is related to an opened issue : #

**⚠️ If changes affect GitHub Action files**

- [ ] I have added tests to cover my changes : [link](url)

## Pull request type

Please check the type of change your PR introduces:

- [x] Documentation content changes
- [ ] Bugfix on the site
- [ ] Build related changes
- [ ] Other (please describe):
      
## Description
Add changelog entry about first part of February images update

### CODE REVIEW

File: content/changelog/2024-02-02-images-update.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: February 2024 images update (part 1)
```
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:2]: replace `images` by `applications` maybe?

File: content/changelog/2024-02-02-images-update.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+date: 2024-02-02
+tags:
+  - images
```
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:5]: _idem_

File: content/changelog/2024-02-02-images-update.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - security
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+description: An in-depth work leading to faster updates
+excludeSearch: true
+---
+
+Over the past few months, we've overhauled the way we build our applications and add-ons images from tooling to included binaries. We'll deliver soon the first public milestone of this project, which will leads us to more frequent releases.
```
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:16]: Remove ` We'll deliver soon the first public milestone of this project, which will leads us to more frequent releases` (no future specs or announcements, just documenting what is effective at the moment of publication). Maybe add `This is the first part of the updated images`, or something like that, instead?

File: content/changelog/2024-02-02-images-update.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+In the meantime, we've updated Docker, Erlang, Go, Haskell, Ruby and Rust images into production with no impact for our users. They include security patches and now use Linux kernel 6.7.1, OpenSSL 3.2.1 and Node.js 20.11.0 by default. For the latter, you can change it in the updated images via the `CC_NODE_VERSION` [environment variable](/doc/reference/reference-environment-variables/#commons-to-all-applications). Other changes are as follows:
```
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:18]:
```suggestion
We've updated Docker, Erlang, Go, Haskell, Ruby and Rust images into production with no impact for our users. They include security patches and now use Linux kernel 6.7.1, OpenSSL 3.2.1 and Node.js 20.11.0 by default. For the latter, you can change it in the updated images via the `CC_NODE_VERSION` [environment variable](/doc/reference/reference-environment-variables/#commons-to-all-applications). Other changes are as follows:
```

File: content/changelog/2024-02-02-images-update.md (hunk 1 above)
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:2]:
```suggestion
title: Applications update: Docker, Erlang, Rust, Ruby, Haskell, Go
```

File: content/changelog/2024-02-02-images-update.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  - elixir 1.15.7, erlang 26.2.1
+  - elixir 1.16.0, erlang 26.2.1
+- **Rust**:
+  - Stable: 1.75.0
+  - Beta: 1.76.0
+  - Nightly: 1.77.0
+- **Go**:
+  - go 1.21.6
+
+Other images will be updated the same way in the coming days.
```
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:35]:
```suggestion

```

File: content/changelog/2024-02-02-images-update.md (hunk 3 above)
Comment (juliamrch) [inline content/changelog/2024-02-02-images-update.md:16]:
```suggestion
Over the past few months, we've overhauled the way we build our applications and add-ons images from tooling to included binaries.
```

**Review by juliamrch — CHANGES_REQUESTED** [review]
That's a big update, I've added some suggestions to increase clarity.

The main point is avoiding promises or future releases in this section and just documenting what's been implemented so far. Otherwise, all good to me.

**Review by cnivolle — COMMENTED** [review]
I agree with Julia's propositions.
Otherwise, this is good to go.

File: content/changelog/2024-02-02-images-update.md (hunk 1 above)
Comment (davlgd) [reply content/changelog/2024-02-02-images-update.md:2]: We've updated images and it's what this entry is about, so I don't think it's a good thing to change this

File: content/changelog/2024-02-02-images-update.md (hunk 3 above)
Comment (juliamrch) [reply content/changelog/2024-02-02-images-update.md:16]: Ok then, it's not doc do let's go

**Pull request merged**